packages/webcomponents/src/components/business-form-stepped/business-form-stepped.tsx

```diff
@@ -63,137 +74,98 @@ export class BusinessFormStepped {
 
-    this.formController = new FormController(businessFormSchema);
-    this.api = Api(this.authToken, config.proxyApiOrigin);
+    this.refs = [this.coreInfoRef, this.legalAddressRef, this.additionalQuestionsRef, this.representativeRef];
     this.totalSteps = Object.keys(this.componentStepMapping).length - 1;
-    this.fetchData();
   }
 
-  handleResponse(response, onSuccess) {
-    if (response.error) {
-      this.serverError = true;
-      this.errorMessage = response.error.message;
-    } else {
-      this.serverError = false;
-      this.businessId = response.data.id;
-      this.formController.setInitialValues(response.data);
-      onSuccess();
-    }
+  handleFormLoading(e: CustomEvent) {
+    this.formLoading = e.detail;
   }
 
-  private async sendData(onSuccess?: () => void) {
-    // Stopgap solution to prevent sending data in storybook examples.
-    if (this.testMode) {
-      onSuccess();
-      return;
-    }
-
-    this.isLoading = true;
-
-    try {
-      const values = this.formController.values.getValue();
-      const initialValues = this.formController.getInitialValues();
-      const payload = parseForPatching(values, initialValues);
-      const response = await this.api.patch(this.businessEndpoint, JSON.stringify(payload));
-      this.handleResponse(response, onSuccess);
-      this.submitted.emit({ data: response.data });
-    } catch (error) {
-      this.serverError = true;
-      this.errorMessage = error.message;
-    } finally {
-      this.isLoading = false;
-    }
+  handleServerErrors(e: CustomEvent) {
+    this.serverError = true;
+    this.errorMessage = e.detail.message;
   }
 
-  private async fetchData() {
-    this.isLoading = true;
-    try {
-      const response: IApiResponse<IBusiness> = await this.api.get(this.businessEndpoint);
-      const business = new Business(response.data);
-      this.formController.setInitialValues(business);
-    } catch (error) {
-      this.serverError = true;
-      this.errorMessage = `Error fetching data: ${error.message}`;
-    } finally {
-      this.isLoading = false;
-    }
+  showPreviousStepButton() {
+    return this.currentStep > 0;
   }
 
-  private validateAndSubmit(event: any) {
-    event.preventDefault();
-    this.formController.validateAndSubmit(this.sendData);
+  showNextStepButton() {
+    return this.currentStep < this.totalSteps;
   }
 
-  showPreviousStepButton() {
-    return this.currentStep > 0;
+  incrementSteps = () => { 
+    if (this.currentStep < this.totalSteps) {
+      return this.currentStep++; 
+    }
   }
+  decrementSteps = () => { return this.currentStep--; }
 
   previousStepButtonOnClick() {
     this.clickEvent.emit({ name: ClickEvents.previousStep })
-    this.sendData(() => this.currentStep--);
-  }
-
-  showNextStepButton() {
-    return this.currentStep < this.totalSteps;
+    const currentStep = this.refs[this.currentStep];
+    currentStep.validateAndSubmit(this.decrementSteps);
```

Comment:
This could be slightly easier to read if we do: 
`currentStep.validateAndSubmit({ onSuccess: this.decrementSteps });`

And then in `business-core-info-form-step.tsx`
```
  @Method()
  async validateAndSubmit({ onSuccess }: { onSuccess: () => Promise<void> }) {
    this.formController.validateAndSubmit(() => this.sendData(onSuccess));
  };
  ```